src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/sulu-content.js

```diff
@@ -155,37 +163,45 @@ define(function() {
          * Handles the content part of the layout object.
          *
          * @param {Object} content The content config object.
+         * @param {Boolean} defaults Iff true default actions will be executed
          */
-        handleLayoutContent = function(content) {
+        handleLayoutContent = function(content, defaults) {
```

Comment:
i think "useDefaukts"/"applyDefaults" would be a better name for the second parameter